docs/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Why the name of the file LICENSE.txt and not just LICENSE

Reply:
@NupurThakur27 Thanks for pointing that out! It needs to be updated/removed. I'll get it done :)